cycle/cycle.env

```diff
@@ -102,6 +102,7 @@ ts_electrometer=0.8.0a1
 ts_fiberspectrograph=0.8.0
 ts_weatherforecast=0.3.1
 ts_gis=0.4.2a1
+robotframework_efd=tickets/DM-36343
```

Comment:
This should be moved to `develop` since that branch has been delete otherwise this won't work anymore.